packages/devextreme/js/__internal/events/utils/index.ts

```diff
@@ -1,196 +1,212 @@
+  x: to.x - from.x,
+  y: to.y - from.y,
+  time: to.time - from.time || 1,
 });
 
-export const hasTouches = e => {
-    const { originalEvent, pointers } = e;
+export const hasTouches = (e) => {
+  const { originalEvent, pointers } = e;
 
-    if(isNativeTouchEvent(e)) {
-        return (originalEvent.touches || []).length;
-    }
+  if (isNativeTouchEvent(e)) {
+    return (originalEvent.touches || []).length;
+  }
 
-    if(isDxEvent(e)) {
-        return (pointers || []).length;
-    }
+  if (isDxEvent(e)) {
+    return (pointers || []).length;
+  }
 
-    return 0;
+  return 0;
 };
 
 // TODO: for tests
 let skipEvents = false;
-export const forceSkipEvents = () => skipEvents = true;
-export const stopEventsSkipping = () => skipEvents = false;
-
-export const needSkipEvent = e => {
-    // TODO: for tests
-    if(skipEvents) {
-        return true;
+export const forceSkipEvents = () => { skipEvents = true; };
+export const stopEventsSkipping = () => { skipEvents = false; };
+
+export const needSkipEvent = (e) => {
+  // TODO: for tests
+  if (skipEvents) {
+    return true;
+  }
+
+  // TODO: this checking used in swipeable first move handler. is it correct?
+  const { target } = e;
+  const $target = $(target);
+  const isContentEditable = target?.isContentEditable || target?.hasAttribute('contenteditable');
+  const touchInEditable = $target.is('input, textarea, select') || isContentEditable;
+
+  if (isDxMouseWheelEvent(e)) {
+    const isTextArea = $target.is('textarea') && $target.hasClass('dx-texteditor-input');
+
+    if (isTextArea || isContentEditable) {
+      return false;
     }
 
-    // TODO: this checking used in swipeable first move handler. is it correct?
-    const { target } = e;
-    const $target = $(target);
-    const isContentEditable = target?.isContentEditable || target?.hasAttribute('contenteditable');
-    const touchInEditable = $target.is('input, textarea, select') || isContentEditable;
-
-    if(isDxMouseWheelEvent(e)) {
-        const isTextArea = $target.is('textarea') && $target.hasClass('dx-texteditor-input');
+    const isInputFocused = $target.is('input[type=\'number\'], textarea, select') && $target.is(':focus');
 
-        if(isTextArea || isContentEditable) {
-            return false;
-        }
+    return isInputFocused;
+  }
 
-        const isInputFocused = $target.is('input[type=\'number\'], textarea, select') && $target.is(':focus');
+  if (isMouseEvent(e)) {
+    return touchInEditable || e.which > 1; // only left mouse button
+  }
 
-        return isInputFocused;
-    }
-
-    if(isMouseEvent(e)) {
-        return touchInEditable || e.which > 1; // only left mouse button
-    }
-
-    if(isTouchEvent(e)) {
-        return touchInEditable && focused($target);
-    }
+  if (isTouchEvent(e)) {
+    return touchInEditable && focused($target);
+  }
 };
 
-export const setEventFixMethod = func => fixMethod = func;
+export const setEventFixMethod = (func) => { fixMethod = func; };
```

Comment:
This part can be considered correct, but only if we are sure that the returned value is not used anywhere